lib/_stream_readable.js

```diff
@@ -1139,3 +1139,31 @@ function endReadableNT(state, stream) {
     }
   }
 }
+
+Readable.from = function(iterator, opts) {
```

Comment:
I think it would make sense to default to objectMode here given that's virtually all the usage I've seen "in the wild" and certainly the more common case :]

Reply:
Makes good sense!